ROADMAP.md

```diff
@@ -2,60 +2,47 @@
-technical, automation, and documentation debt. Please take the timelines and dates as proposals and goals.
-Priorities and requirements change based on community feedback, roadblocks encountered, community contributions,
-etc. If you depend on a specific item, we encourage you to attend community meetings to get updated status information,
-or help us deliver that feature by contributing to Tanzu Community Edition.
-
-Last Updated: October 2021
-
-|Theme|Feature|Timeframe|
-|--|--|--|
-|**Build user community**|Start public Office Hours and Community Meetings|0-3 months|
-||Start Slack channel and Google Group|0-3 months|
-||Training/enablement resources (including KubeAcademy and Tanzu Developer Center|0-3 months|
-||Reward community support|3-6 months|
-||Community roadmap prioritizatiion|3-6 months|
-|**Accelerate time to value**|Include demo applications for more complete startup experience|3-6 months|
-||Work to reduce time to cluster availability on vSphere and public clouds|3-6 months|
-||Pre-integrated multi-package solutions|3-6 months|
-|**Robust Kubernetes platform**|Limited-internet (including airgapped) support|0-3 months|
-||Aligned to upstream Kubernetes releases|0-3 months|
-||Early availability of upstream Kubernetes releases|3-6 months|
-|**Rich package library**|Work with community to solicit and prioritize desired packages|0-3 months|
-||Pursue community and ISV package lifecycle and configuration integrations via Carvel and Tanzu Framework|3-6 months|
-|**Tanzu scenario support beyond Kubernetes**|Tanzu Application Platform OSS components|0-3 months|
-||Add developer focused UI experience|0-3 months|
-||Bitnami integration via kubeapps|0-3 months|
+The following table includes the current roadmap for Tanzu Community Edition. If you have any questions or would like to contribute, please attend a community meeting to discuss with our team. If you don't know where to start, we are always looking for contributors that will help us reduce technical, automation, and documentation debt. Please take the timelines and dates as proposals and goals. Priorities and requirements change based on community feedback, roadblocks encountered, community contributions, etc. If you depend on a specific item, we encourage you to attend community meetings to get updated status information, or help us deliver that feature by contributing to Tanzu Community Edition.
+
+Last Updated:  January 2022
+
+| Theme | Feature | Status (Phase) | Targeted Release |
+| ----- | ------- | -------------- | ---------------- |
+| Build user community | Reward community support | n/a | ongoing |
+| |Community roadmap prioritization | n/a | ongoing |
+| Enhance cloud-native user experience for skilled and new users | Unmanaged-cluster model introduced to enable minimal-cluster deployments in under 4 minutes that can run on consumer hardware. | Implementation in Progress | v0.11 |
```

Comment:
:+1: really love this one. (just positive feedback, feel free to close)

Reply:
Do we want it to say v0.11? The current plan is to include it in the v0.10.0 release, correct?

Reply:
Un-resolved this one since it still looks like it's marked for v0.11?

Reply:
Unmanaged now says 0.10.